packages/react/src/VideoPlayer/components/PauseIcon/PauseIcon.tsx

```diff
@@ -0,0 +1,8 @@
+import React, {type SVGAttributes} from 'react'
+
+export const PauseIcon = (props: SVGAttributes<SVGSVGElement>) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" {...props}>
```

Comment:
Is this an Octicon @joshfarrant? If so, can we use the one from the `@primer/octicons-react` package here instead of  hardcoding our own?

Reply:
I'm not sure where this came from tbh, it's just moved over from what the VideoPlayer was using before. I'll have a look

Reply:
Ah yea now I remember, there isn't a simple _Play_ Octicon and there's no pause one at all, hence why these were made in the first place.

Reply:
The official one is this https://primer.style/foundations/icons/play-24

I think we should keep it custom for now, and follow up with a request to Octicons working group about a play icon that doesn't have the rounded border. In long term, we don't want to maintain any of our own icons.

cc. @simmonsjenna @danielguillan for second opinions... the official Octicon has a rounded border, so quite different to custom one used in video player right now. Let us know if you have have a preference.